metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -111,8 +115,8 @@ object MetalsEnrichments
   }
 
   implicit class XtensionCompileResult(result: b.CompileResult) {
-    def isOK: Boolean = result.getStatusCode.isOK
-    def isError: Boolean = result.getStatusCode.isError
+    def isOK: Boolean = result.getStatusCode == b.StatusCode.OK
```

Comment:
I think this might have changed due to resolution of merge conflicts? Could you revert it?

Reply:
@tgodzik I rebased onto main for resolving the conflict and I guess I picked the wrong resolution! I should have picked the main one…
I didn’t pay attention and thought that the difference was just the white space!

Reply:
No worries, that's why it's usually a good idea to take a look at the PR itself again to double check. I usually do that especially after rebasing with conflicts.

Reply:
solved! And yes, I need to take a look at the PR after rebase...